### PR TITLE
[SD-2413] and [SD-2414] Add desk/stage to version history and fix by bug

### PR DESCRIPTION
**Pull request by marwoodandrew**

### CODE REVIEW

**Comment by sivakuna-aap** [comment]
:+1:

File: client/app/scripts/superdesk-authoring/versioning/versions.js [hunk 1]
```diff
@@ -2,42 +2,52 @@
...
-            _.each(result._items, function(version) {
-                var creator = version.creator || version.original_creator;
-                if (creator && !$scope.users[creator]) {
-                    fetchUser(creator);
-                }
+        desks.initialize()
+            .then(function() {
+                $scope.desks = desks.desks;
+                $scope.stages = desks.deskStages;
+                $scope.users = desks.users;
```
Comment (petrjasek) [inline client/app/scripts/superdesk-authoring/versioning/versions.js]: seems like `$scope.desks`, `$scope.stages` and `$scope.users` are not needed anymore

**Comment by petrjasek** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

**Pull request merged**